website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+## Focus on Accessibility
+
+There have been many improvements to the accessibility APIs, like `announceForAccessibility`, plus improvements to roles, action support, flags, and more. Accessibility is a complex science, but we hope these improvements make it a bit easier to be an A11Y. Be sure to check [React Native Open Source Update June 2019](https://facebook.github.io/react-native/blog/2019/06/12/react-native-open-source-update#meaningful-community-contributions) for more details of these changes.
+
+## A Fresh Start
+
+React Native's start screen has been updated! Thank you to the many contributors who helped create the new UI. This new "Hello World" will welcome users to the ecosystem in a more friendly, engaging way.
+
+![The new init screen helps developers get started from the get-go with resources and a good example](/react-native/blog/assets/0.60-new-init-screen.png)
+
+## AndroidX Support
+
+[AndroidX](https://developer.android.com/jetpack/androidx) is a major step forward in the Android ecosystem, and the old support library artifacts are being deprecated. For 0.60, React Native has been migrated over to AndroidX. This is a breaking change, and **your native code and dependencies will need to be migrated** as well.
+
+> With this change, React Native apps will need to begin using AndroidX themselves. They cannot be used side-by-side in one app, so all of the app code and dependency code needs to be using one or the other.
+>
+> [matt-oakes](https://github.com/matt-oakes) on [discussions-and-proposals](https://github.com/react-native-community/discussions-and-proposals/issues/129)
+
+While your own native code will need to be migrated by you, [@mikehardy](https://github.com/mikehardy), [@cawfree](https://github.com/cawfree), and [@m4tt72](https://github.com/m4tt72) built a [clever tool named "jetifier"](https://github.com/mikehardy/jetifier) to patch your `node_modules`. Library maintainers will need to upgrade, but this tool will help while waiting. So if you find errors related to AndroidX migration, give this a shot.
+
+## CocoaPods by Default
+
+[CocoaPods are now part of React Native's iOS project](https://github.com/react-native-community/discussions-and-proposals/blob/master/proposals/0004-cocoapods-support-improvements.md). If you weren't already, be sure to open iOS platform code using the `xcworkspace` file from now on. Also, the `podspec`s for the internal packages have changed to make them compatible with the Xcode projects, which will help with troubleshooting and debugging. Expect to make some straightforward changes to your `Podfile` as part of the upgrade to 0.60 to bring this exciting support.
```

Comment:
What do you think about mentioning this content as a tip: https://twitter.com/lbentosilva/status/1143055879664783360?s=20?

Reply:
I wonder if we should mention this https://github.com/react-native-community/releases/pull/120#issuecomment-507305472 here too

Reply:
Would be nice 😁